api/src/main/java/org/openmrs/module/locationbasedaccess/PatientSearchAdviser.java

```diff
@@ -39,6 +40,9 @@ public boolean matches(Method method, Class targetClass) {
         else if (method.getName().equals("getPatient")) {
             return true;
         }
+        else if (method.getName().equals("getPatientByUuid")) {
```

Comment:
Don't we already have this here? https://github.com/openmrs/openmrs-module-locationbasedaccess/blob/master/api/src/main/java/org/openmrs/module/locationbasedaccess/PatientSearchAdviser.java#L42